extflag/internal/timestamp/timestamp.go

```diff
@@ -0,0 +1,28 @@
+// Taken from "github.com/prometheus/prometheus/pkg/timestamp"
+//
+// Copyright 2017 The Prometheus Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package timestamp
+
+import "time"
+
+// FromTime returns a new millisecond timestamp from a time.
+func FromTime(t time.Time) int64 {
+	return t.Unix()*1000 + int64(t.Nanosecond())/int64(time.Millisecond)
+}
```

Comment:
simpler to do `return t.UnixNano()  / 1e6 `?

Reply:
Thanks! Also will finish tests today. Thanks for looking! :muscle: 

Please let me know if you have any other feedback to the library.

Reply:
Hm, I copied it over Prometheus so let's maybe stick to this for now :thinking: